cmd/syft/internal/test/integration/package_binary_elf_relationships_test.go

```diff
@@ -0,0 +1,66 @@
+package integration
+
+import (
+	"github.com/anchore/syft/syft/artifact"
+	"github.com/anchore/syft/syft/source"
+	"github.com/stretchr/testify/require"
+	"testing"
+)
+
+func TestBinaryElfRelationships(t *testing.T) {
+	// node --> ["dependency of" nodes]
+	expectedGraph := map[string][]string{
+		"glibc": {
+			"libhello_world.so",
+			"syfttestfixture",
+		},
+		"libstdc++": {
+			"syfttestfixture",
+		},
+		"libhello_world.so": {
+			"syfttestfixture",
+		},
+		// TODO: we need to capture devel package
```

Comment:
is this stale? or defunct?

Reply:
This is defunct now we need to test against this. I'l remove the old comment